.travis.yml

```diff
@@ -21,11 +21,5 @@ script:
   - sleep 5
   - pylinkvalidate.py http://localhost:8001/
   - kill "${pid}"
-  - |
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-      # gets commit message of last commit before pull request merge
-      COMMIT_MESSAGE=$(git log $TRAVIS_PULL_REQUEST_SHA --format=%B -n 1)
-      printf "Checking commit message:\n\n"
-      printf "$COMMIT_MESSAGE\n\n"
-      checkcommit --message "$COMMIT_MESSAGE"
-    fi
+  - openwisp-utils-qa-checks --skip-checkmigrations --skip-isort --skip-flake8 --skip-checkmakemigrations --skip-checkendline
```

Comment:
why are we skipping `--skip-checkendline`? Is there some file which doesn't have the ending line and which should be kept as it is right now?
That check would be useful.